examples/anime-http/server/asyncapi.yaml

```diff
@@ -53,3 +58,17 @@ components:
         studio:
           type: string
           description: The studio of anime.
+  securitySchemes:
+    tokens:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+    userPass:
+      type: userPassword
+    apiKey:
+      type: httpApiKey
+      name: api_key
+      in: header
+    cert:
+      type: apiKey
+      in: user
```

Comment:
We definitly documentation to explain how each of these work with glee and show some snippets of code on how to set it up and do the authentecation process for each one.

Reply:
> We definitly documentation to explain how each of these work with glee and show some snippets of code on how to set it up and do the authentecation process for each one.

Could you explain further Khuda, it's kinda confusing @KhudaDad414

Reply:
I meant we should have a documentation for the types of auths that can be done here. sorry didn't see your comment. 🤦